packages/build/src/time/aggregate.js

```diff
@@ -31,7 +31,7 @@ const isTotalTimer = function ({ stageTag, parentTag }) {
 const createOthersTimer = function (topTimers, { durationNs: totalTimerDurationNs }) {
   const topTimersDurationNs = computeTimersDuration(topTimers)
   const otherTimersDurationNs = Math.max(0, totalTimerDurationNs - topTimersDurationNs)
-  const othersTimer = createTimer(OTHERS_STAGE_TAG, otherTimersDurationNs)
+  const othersTimer = createTimer(OTHERS_STAGE_TAG, otherTimersDurationNs, { tags: { success: true } })
```

Comment:
Wasn't sure here. Right now the `other` stage is always successful. I could change that to reflect the overall state? Like when anything fails then other is also set to `success:false` ? WDYT?

Reply:
Thinking it makes sense to keep at `success: true` 🤔 .